tmk_core/protocol/ps2_interrupt.c

```diff
@@ -40,8 +40,16 @@ POSSIBILITY OF SUCH DAMAGE.
  */
 
 #include <stdbool.h>
+
+#if defined(__AVR__)
 #include <avr/interrupt.h>
 #include <util/delay.h>
```

Comment:
I think this one can also be replaced with `#include "wait.h` but outside the avr check, and maybe down by the `print.h` include.

Reply:
```suggestion
```